pandas/__init__.py

```diff
@@ -190,6 +192,21 @@
     __git_version__ = v.get("full-revisionid")
     del get_versions, v
 
+# GH#55043 - if `import pandas` is using ArrayManager, user has env variable set
+from pandas._config.config import _get_option
+
+if _get_option("mode.data_manager", silent=True) == "array":
```

Comment:
does any of this affect the import time `python -X importtime -c "import pandas as pd"`?

Reply:
Reporting cumtime for pandas

 - main: 609562, 608411, 616030
 - PR: 605456, 611654, 609585

Reply:
Ah - I should have mentioned - I ran without any env variables set. I expect that to be by far the common case.

Reply:
Latest PR: 593762, 607619, 620866